README.md

```diff
@@ -0,0 +1,2 @@
+Please note that development of l10nregistry has moved to
+[mozilla-central](https://hg.mozilla.org/mozilla-central/file/tip/intl/l10n/rust/l10nregistry-rs).
```

Comment:
It would be good to link to:
 * File a bug - https://bugzilla.mozilla.org/enter_bug.cgi?product=Core&component=Internationalization
 * See open bugs - https://bugzilla.mozilla.org/buglist.cgi?product=Core&component=Internationalization&bug_status=__open__&list_id=16008019